src/kvstore/test/RocksEngineTest.cpp

```diff
@@ -19,23 +19,30 @@ namespace kvstore {
 
 const int32_t kDefaultVIdLen = 8;
 
-class RocksEngineTest : public ::testing::TestWithParam<std::tuple<bool, bool, std::string>> {
+class RocksEngineTest : public ::testing::TestWithParam<std::tuple<bool, bool, std::string, bool>> {
  public:
   void SetUp() override {
     auto param = GetParam();
     FLAGS_enable_rocksdb_prefix_filtering = std::get<0>(param);
     FLAGS_enable_rocksdb_whole_key_filtering = std::get<1>(param);
     FLAGS_rocksdb_table_format = std::get<2>(param);
+    flush_ = std::get<3>(param);
   }
 
   void TearDown() override {}
+
+ protected:
+  bool flush_;
 };
 
 TEST_P(RocksEngineTest, SimpleTest) {
   fs::TempDir rootPath("/tmp/rocksdb_engine_SimpleTest.XXXXXX");
   auto engine = std::make_unique<RocksEngine>(0, kDefaultVIdLen, rootPath.path());
   EXPECT_EQ(nebula::cpp2::ErrorCode::SUCCEEDED, engine->put("key", "val"));
   std::string val;
+  if (flush_) {
```

Comment:
Why is flush needed?

Reply:
Because, we need to verify the result is correct no matter data is in memtable or sst